src/js/utils/JobUtil.js

```diff
@@ -90,6 +90,9 @@ const JobUtil = {
 
     let [schedule = {}] = job.getSchedules();
     if (schedule.id != null && schedule.cron != null) {
+      if (schedule.concurrencyPolicy == null) {
+        schedule.concurrencyPolicy = 'ALLOW';
```

Comment:
That's bad. Have we also double-checked that `ALLOW` is the most sensible default?